src/pages/dashboard/HomePage.tsx

```diff
@@ -0,0 +1,160 @@
+            if (queryString) {
+              newSearchParams.set(SearchParam.Query, queryString);
+            }
+            newSearchParams.set(SearchParam.Results, previousParamsResults ?? defaultResultSize);
+            newSearchParams.set(SearchParam.From, '0');
+          } else if (personIsSeleced) {
+            newSearchParams.set(SearchParam.Type, SearchTypeValue.Person);
+            if (values.searchTerm) {
+              newSearchParams.set(SearchParam.Name, values.searchTerm);
+              newSearchParams.set(SearchParam.Results, previousParamsResults ?? defaultResultSize);
+              newSearchParams.set(SearchParam.Page, '1');
+            }
+          } else if (projectIsSelected) {
+            newSearchParams.set(SearchParam.Type, SearchTypeValue.Project);
+            if (values.searchTerm) {
+              newSearchParams.set(SearchParam.Query, values.searchTerm);
+              newSearchParams.set(SearchParam.Results, previousParamsResults ?? defaultResultSize);
+              newSearchParams.set(SearchParam.Page, '1');
+            }
+          }
+          history.push({ search: newSearchParams.toString() });
+        }}>
+        {({ resetForm }: FormikHelpers<SearchConfig>) => (
+          <Form style={{ width: '100%' }}>
+            <StyledPageWithSideMenu>
+              <SideMenu>
+                <SideNavHeader icon={SearchIcon} text={t('common.search')} />
+                <NavigationListAccordion
+                  title={t('common.filter')}
+                  startIcon={<FilterIcon sx={{ bgcolor: 'white' }} />}
+                  accordionPath=""
+                  expanded={isOnSearchPage}
+                  dataTestId={dataTestId.startPage.filterAccordion}>
+                  <>
+                    {resultIsSelected && searchResults?.aggregations ? (
+                      <Box
+                        sx={{
+                          m: '1rem',
+                          display: 'flex',
+                          flexDirection: 'column',
+                          gap: '1rem',
+                        }}>
+                        <RegistrationFacetsFilter
+                          aggregations={searchResults.aggregations}
+                          isLoadingSearch={isLoadingSearch}
+                        />
+                      </Box>
+                    ) : (
+                      <Typography fontStyle="italic" sx={{ mx: '1rem', mb: '1rem' }}>
+                        {t('search.no_available_filters')}
+                      </Typography>
+                    )}
+                  </>
+                </NavigationListAccordion>
+
+                <NavigationListAccordion
+                  title={t('search.reports.reports')}
+                  startIcon={<InsightsIcon sx={{ bgcolor: 'white' }} />}
+                  accordionPath={UrlPathTemplate.Reports}
+                  dataTestId={dataTestId.startPage.reportsAccordion}
+                  onClick={() => resetForm()}>
```

Comment:
Ser ikke ut som resetForm virker helt eller? ser fortsatt bugen hos meg når jeg navigrer mellom filter og rapporter

Reply:
For å reprodusere må du åpne siden direkte enten via link eller som refresh, feks: http://localhost:3000/?query=%22kapittel%22+AND+(topLevelOrganization.id%3D%22https%3A%2F%2Fapi.dev.nva.aws.unit.no%2Fcristin%2Forganization%2F20754.0.0.0%22)+AND+(entityDescription.contributors.identity.id%3A%22https%3A%2F%2Fapi.dev.nva.aws.unit.no%2Fcristin%2Fperson%2F1136254%22)&results=10&from=0

Når man da klikker på "Rapporter", og så "Filter" igjen vil filtrene fortsatt være der, men de vil ikke brukes